lib/uploaded-file.js

```diff
@@ -13,14 +13,18 @@ module.exports = class UploadedFile {
     this.name = null;        // file name
     this.contentType = null; // downloaded content type
 
-    // audio metadata
-    this.duration  = null; // in seconds, ROUNDED UP
-    this.size      = null; // in bytes
-    this.format    = null; // codec name
-    this.bitrate   = null; // 64/128/256
-    this.frequency = null; // float 44.10 / 48.00 / etc
-    this.channels  = null; // number of channels
-    this.layout    = null; // mono/stereo/etc
+    // audio/video metadata
+    this.hasAudio    = null;
+    this.duration    = null; // in seconds, ROUNDED UP
+    this.durationMs  = null; // in milliseconds
```

Comment:
just a nice to have? or you lookin at https://github.com/PRX/cms.prx.org/issues/314 ?

Reply:
Both!  In the spirit of not generalizing data here ... will leave it up to CMS to do that in the future.

Reply:
👍